### ISSUE TITLE
fix handling of servables published with the same shortname and title

### ISSUE THREAD

**Issue by ascourtas** [DLHub-Argonne/dlhub_sdk#151]
Currently, when a user publishes two models with the same shortname and title, the SDK is supposed to amend the shortname (i.e. with a version number) to differentiate it from the previously published model. 

Currently, that does not happen and both models are published with the exact same shortname and title 
https://acdc.alcf.anl.gov/dlhub/?q=braggnn_pt_cpu 

We need to fix the handling logic at publication time to make sure we don't have duplicate shortnames/titles

**Comment by WardLT** [comment]
Is the name not changing actually a problem? We use the version number to disambiguate the most recent version of a servable

**Comment by WardLT** [comment]
The DLHubClient sorts by publication date to get the latest version each time we access a model by its short name

**Comment by ascourtas** [comment]
Per Slack conversations, we realized there was a lot of confusion about what was needed for this issue. 

Ultimately, there is "versioning" of sorts in DLHub currently in the form of the publication date -- this is not necessarily intuitive to the user, but is currently critical for how the client gets the most recent version of the model. The MDF way of versioning is also not super desired. 

One suggestion was to have the version in the metadata.

We decided to hold off on this until our Garden rewrite to decide on the best design.
